### PR TITLE
Fix for undefined Modernizr

### PR DESCRIPTION
**Pull request by sergeome**
Pull request for [this issue](https://github.com/MotifFramework/Motif/issues/14).

### CODE REVIEW

File: js/actions.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * Sample actions execution file
  */
 
-(function ( $, window, document, Motif, undefined ) {
+(function ( $, window, document, Modernizr, Motif, undefined ) {
```
Comment (frontendauteur) [inline js/actions.js]: This removes the undefined, but it just assigns it to the 4th parameter on Line 46, which is `window.Motif`. I don't think this is complete until we add `window.Modernizr` (or something) as the 4th parameter at the bottom.

**Comment by sergeome** [comment]
You're right. Sorry, that doesn't make sense at all without a parameter.
After adding parameter I've faced with another issue (but it should work in this way): Browser still throws an error (Please see "Undefined" attachment for more details).

undefined


I moved "vendor/modernizr.js" declaration from `helpers` to `main` in 'uglify.js'. It's working with this approach. Please see "Modernizr object" for more details. 

modernizr object


But I'm not sure if it's a real fix or just a hack. Could you please suggest?

**Comment by frontendauteur** [comment]
@sergeome I think the actual issue is that you're expected to include `helpers.motif.js` in the `<head>` and `motif.js` at the end of the `<body>`. From the looks of #14 I think you're missing `helpers.motif.js`, therefore missing `Modernizr`. So technically, yes, moving it to the `main` uglify build fixes this one issue, I think we are now solving something that maybe didn't need solving.

I still think adding `Modernizr`/`window.Modernizr` as parameters of this `actions.js` function is needed, so I want to keep that. Maybe the real issue is unclear instructions/intent.

**Comment by sergeome** [comment]
@frontendauteur,

I've added helpers.motif.js inside the head before "motif.js" and problem has disappeared. 
Thank you for pointing me to that!
So, it's not a bug. It's inappropriate configuration from my side.

I think I can close this PR and related bug I reported.

**Pull request closed without merging**